package.json

```diff
@@ -1,6 +1,9 @@
 {
   "private": true,
   "name": "o-topper",
+  "dependencies": {
+    "@financial-times/n-map-content-to-topper": "^1.1.0"
+  },
```

Comment:
```suggestion
```

we don't need this here because it's added by [occ](https://github.com/Financial-Times/origami-component-converter/) during release :)

Reply:
great! I will remove 😄